style2.css

```diff
@@ -0,0 +1,145 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  max-width: 375px;
+  height: 50px;
+}
+
+.welcome {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: semi bold;
+  font-size: 20px;
+  line-height: 28px;
+  margin-left: 35px;
+  display: inline-block;
+}
+
+.union {
+  width: 18px;
+  height: 12px;
+  display: inline-block;
+  margin-left: 230px;
+}
+
+.flex {
+  max-width: 375px;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] It's nice to give this classname to flex container, but in order to make your code DRY, please use this class for every flex containers and add `display: flex;`  in it so that the name can have more meaning.